lib/src/app/app.dart

```diff
@@ -164,4 +164,13 @@ class AppClient {
       ..userId = userId;
     await _client.deleteOrganizationMember(request);
   }
+
+  /// Create a new smart machine with the included [name] in the passed in [locationId]
+  Future<String> newMachine(String name, String locationId) async {
```

Comment:
debated calling this `newRobot` to be more transparent with the underlying proto - but it seems like that would be changing soonish and would become a breaking change. Trying to mitigate the amount of those in the future. But I can also see the argument with staying consistent and doing a hard cutover in the future.